frontend/src/utils/video.ts

```diff
@@ -167,8 +167,15 @@ export async function getVideoForComparison(
     );
     if (videoFromComparisons) return videoFromComparisons;
   }
-  const videoResult = await VideoService.videoList({ limit: 100, offset: 0 });
-  const videoList = (videoResult?.results || []).map((v) => v.video_id);
+  const videoResult = await PollsService.pollsRecommendationsList({
+    name: 'videos',
+    limit: 100,
+    offset: 0,
+  });
+  // console.log(videoResult);
+  const videoList = (videoResult?.results || []).map((v) =>
+    v.uid.slice(3, v.uid.length)
```

Comment:
@GresilleSiffle Is there a more robust way to get the youtube ids? Maybe even returning the Tournesol uid could work here?

Reply:
In the code snippet `v.uid.slice(3, v.uid.length)` I am taking out the youtube ids from the Tournesol uid itself. The `video_id` was not given out by the model defined in `/frontend/src/services/openapi/models/Recommendation.ts`.

If the video_id and uid are always similar with just a `yt:` add-on difference, we can just use the `uid` and avoid redundancy, if possible.

Reply:
> @GresilleSiffle Is there a more robust way to get the youtube ids? Maybe even returning the Tournesol uid could work here?

Yes, we have the function `idFromUid` that does exactly what we want.

> Maybe even returning the Tournesol uid could work here?

Yes, but not as-is. We will need to adapt the component `<AutoEntityButton>` which is expecting a YouTube id and maybe also the functions taking the variable `videoList` as parameter to be sure we don't break anything. I think we can limit the scope of this PR to the update of the API endpoint, and do the "youtube id to uid" cleaning just after.